charts/firecloudui/README.md

```diff
@@ -1,8 +1,11 @@
-firecloudui
-===========
+# firecloudui
 
 Chart for Firecloud UI service in Terra
 
+## Values
 
-
-
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| global.applicationVersion | string | `"latest"` | What version of the firecloudui application to deploy |
```

Comment:
have you tried using [helm-docs](https://github.com/norwoodj/helm-docs) to programmatically generate your readme.md?

Reply:
Yes, I need to do this! I noticed the README.md file was not up to date when I was reviewing the PR, but didn't have time to do it before I left for my 1p errand. Will do it before I merge this.

Reply:
Personally I just have my local copy of this repo set to run `helm-docs` as a pre-commit hook because I forgot to run it all the time.